apps/docs/content/tokens/semantic/dimensions.mdx

```diff
@@ -8,4 +8,46 @@ import tokens from '../../../datas/tokens.json';
 
 export const categoryKey = "size";
 
-<Table category={categoryKey} data={tokens.semantic[categoryKey]} />
+## Padding
```

Comment:
ya I wondered about this... I could just remove padding and margin and keep the other categories? I just thought something like two headers to summarize it would make it clearer.